Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# only items from loaded addons get pushed into forbiddenItems list

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Extended the logic of creating the A3U_forbiddenItems list so that only classnames from loaded addons get pushed into it.
The new algorythm requires addons[] to be specified for a classname that needs to be blacklisted but is also fully compatible with the old code. (In case someone redefind forbidden_limited_base and/or forbidden_unlimited_base in his extension)

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Load into the game with addons that you have blacklists for and check if the get pushed to A3U_forbiddenItems.
Additionaly check rpt for `Skipped _x from adding to forbiddenItems list.`

### Notes
I used addon names mentioned in trader config to bring existing cfgForbideItems into compliance with new logic. If those addon names are correct there should be no issues.
